proc_dash/app.py

```diff
@@ -128,9 +160,7 @@
                                     children=util.construct_legend_str(
                                         util.PIPE_COMPLETE_STATUS_SHORT_DESC
                                     ),
-                                    style={
```

Comment:
Hum, is this a manual change or a result of your local linter? I think the current pre-commit config may revert these changes - I can probably resolve the resulting merge conflicts with my other open PR manually, but wondering if this would be a simple fix as running `pre-commit install` and `pre-commit run` within your clone.

Another +1 for pre-commit CI, I guess. 🙂 In the meantime, I'll add a local dev note to the README to install the pre-commit hooks.

Reply:
Uhoh, I believe you are correct. I just ran black without any regard for existing rules :| 

I'll revert these changes

Reply:
Thank you thank you!

Reply:
btw: learned something in doing that: `git checkout -p` lets you interactively go chunk (hunk?) by chunk through a previous commit and decide whether to apply it. Very nice. Will add to docs